roofit/multiprocess/test/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 
 add_library(RooFit_multiprocess_testing_utils INTERFACE)
 target_link_libraries(RooFit_multiprocess_testing_utils INTERFACE RooFitCore RooBatchCompute)
-target_include_directories(RooFit_multiprocess_testing_utils PRIVATE ${RooFitMultiProcess_INCLUDE_DIRS})
+target_include_directories(RooFit_multiprocess_testing_utils INTERFACE ${RooFitMultiProcess_INCLUDE_DIRS})
```

Comment:
This in combination of adding the variable to the cache will make external projects compiling against ROOT to try to add the (potentially already deleted) build directory of ROOT to the include paths.

Reply:
See conversation above.